File: MyStagePass.Services/Database/Performer.cs
namespace MyStagePass.Services.Database
{
	public class Performer
	{
		public int PerformerID { get; set; }
		public string? ArtistName { get; set; }
		public string? Bio { get; set; }
		public int UserID {  get; set; }
		public virtual User User { get; set; } = null!;
		public virtual ICollection<PerformerGenre> Genres { get; set; } = new List<PerformerGenre>();
		public virtual ICollection<Event> Events { get; set; } = new List<Event>();
		public bool? IsApproved { get; set; } = null;
	}
}

File: UI/mystagepass_mobile/lib/models/Performer/performer.dart
import 'package:json_annotation/json_annotation.dart';
import '../User/user.dart';

part 'performer.g.dart';

@JsonSerializable()
class Performer {
  int? performerID;
  bool? isApproved;
  String? artistName;
  String? bio;
  List<String>? genres;
  User? user;

  Performer(
    this.performerID,
    this.isApproved,
    this.user,
    this.artistName,
    this.bio,
    this.genres,
  );

  factory Performer.fromJson(Map<String, dynamic> json) =>
      _$PerformerFromJson(json);
  Map<String, dynamic> toJson() => _$PerformerToJson(this);
}
